build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -82,3 +82,6 @@ clickHouse:
   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
   # The minimum interval is 1s based on ClickHouse documentation for best performance.
   #commitInterval: "8s"
+
+  # ttl is the interval to delete flow records from the database
```

Comment:
I just noticed that TTL is configurable in Theia: https://github.com/antrea-io/theia/pull/15
I am not sure this is a property of the FA, and I don't think we should be able to configure the TTL in 2 places.
cc @yanjunz97

Reply:
I see the concern. I think it is added in FA just to have a way to update TTL in runtime. But from the persistence perspective, we should not configure it in FA. Let's remove it for now, and maybe we can take a look in the future whether we can put it into the Theia CLI.

Reply:
I see. I'll remove the ttl related settings.